### DISCUSSION TITLE
Graalvm How to debug remotely

### DISCUSSION THREAD

**Discussion by ruanzhi** [oracle/graal#3196] [Q&A]
https://www.graalvm.org/tools/chrome-debugger/  This article introduces how Graalvm performs remote debugging, but what I want to achieve is: I use java to run graal.js and execute other js scripts, how to code remote debugging, I hope there is a tutorial

**Comment by themartynroberts** [comment]
FWIW I managed to do this by seeing how it is done in the ecplise dirigible project: https://github.com/eclipse/dirigible

It's a moving target but, at the time of writing, here are the relevant parts:
https://github.com/eclipse/dirigible/blob/270b1ce30c722aa42963ae5bef8c3b13edbd8e65/modules/engines/engine-graalium/execution/src/main/java/org/eclipse/dirigible/graalium/core/graal/EngineCreator.java#L34
https://github.com/eclipse/dirigible/blob/270b1ce30c722aa42963ae5bef8c3b13edbd8e65/modules/engines/engine-graalium/execution/pom.xml#L25
